### PR TITLE
debian: package atomic-kms

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -404,6 +418,7 @@ Architecture: linux-any
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${misc:Depends},
+         mir-platform-graphics-atomic-kms,
```
Comment (RAOF) [inline debian/control:421]: Hm. But you haven't actually *added* a `mir-platform-graphics-atomic-kms` package, just a `mir-platform-graphics-atomic-kms22` package.

Maybe what we should do is add `mir-platform-graphics-atomic-kms22` to the dependencies of the `mir-platform-graphics-gbm-kms` metapackage?

**Review by RAOF — CHANGES_REQUESTED** [review]

File: debian/control (hunk 1 above)
Comment (Saviq) [reply debian/control:421]: And then we'll have a `Provides:` / `Breaks:` / `Replaces:`, when we want to move over?

We should probably add `-atomic-kms` already anyway.

Comment (Saviq) [reply debian/control:421]: > We should probably add `-atomic-kms` already anyway.

Done.

**Review by RAOF — CHANGES_REQUESTED** [review]
>  Unknown package: debian/control contains versioned package mir-platform-graphics-atomic-kms but it is unknown to this script

We'll need to fix this before it can land :wink:

File: debian/control [hunk 2]
```diff
@@ -416,6 +431,22 @@ Description: Display server for Ubuntu - desktop driver metapackage
  This package depends on a full set of graphics and input drivers for traditional desktop
  systems.
 
+Package: mir-platform-graphics-atomic-kms
+Section: libs
+Architecture: linux-any
+Multi-Arch: same
+Pre-Depends: ${misc:Pre-Depends}
+Depends: ${misc:Depends},
```
[the remark this answers is not in the document]
Comment (Saviq) [reply debian/control:439]: Or split the rendering platform out?

[the remark this answers is not in the document]
Comment (Saviq) [reply debian/control:440]: I actually added a `Depends` between in `atomic-kms22` on `gbm-kms22`, going through the metapackage felt wrong.

**Review by AlanGriffiths — APPROVED** [review]
Yeah, looks fine now

**Pull request merged**